api/lib/domain/models/CertificationAssessment.js

```diff
@@ -45,6 +45,11 @@ class CertificationAssessment {
     return _.find(this.certificationChallenges, { challengeId }) || null;
   }
 
+  getCertificationAnswersForSkills(skillIds) {
+    const certificationChallenges = _.filter(this.certificationChallenges, (certificationChallenge) => _.includes(skillIds, certificationChallenge.associatedSkillId));
+    const challengeIds = _.map(certificationChallenges, 'challengeId');
```

Comment:
**nitpick:** Je me demande si ce n'est pas plus lisible en enchainant:
```suggestion
  getCertificationAnswersForSkills(skillIds) {
    const challengeIds = this.certificationChallenges
          .filter(({associatedSkillId}) => _.includes(skillIds,associatedSkillId))
          .map(({challengeId}) => challengeId);
```

Reply:
il existe justement différentes préférences là-dessus au sein de l'équipe. Je préfère aussi en enchaînement mais les reviews en équipe récentes m'ont appris qu'une majorité de l'équipe préfère en découpé

Reply:
Perso, j'aime bien décomposé comme ça :)

Reply:
Je trouve que c'est surtout les lignes très longues qui rendent ce morceau de code difficile à lire. Si on veut rester en découpé, ça pourrait être plus lisible comme ça :

```suggestion
const certificationChallenges = this.certificationChallenges.filter(
  (certificationChallenge) => _.includes(skillIds, certificationChallenge.associatedSkillId)
);
const challengeIds = _.map(certificationChallenges, 'challengeId');
```